src/emittermixin.js

```diff
@@ -150,14 +156,18 @@ const EmitterMixin = {
 		//     ...
 		// }
 
-		if ( !( emitters = this._listeningTo ) ) {
-			emitters = this._listeningTo = {};
+		if ( !this[ _listeningTo ] ) {
+			this[ _listeningTo ] = {};
 		}
 
-		if ( !( emitterId = emitter._emitterId ) ) {
-			emitterId = emitter._emitterId = uid();
+		emitters = this[ _listeningTo ];
+
+		if ( !emitter._emitterId ) {
+			emitter._emitterId = uid();
```

Comment:
Another reason to introduce getter/setter is that we need it in `ProxyEmitter`, of course. Earlier I had `_setEmitterId` and `_getEmitterId` methods. Using them would be perhaps more clear, but it's silly to have such methods when those should be getters.